homeassistant/components/amazon_polly/tts.py

```diff
@@ -202,39 +124,48 @@ def get_engine(hass, config, discovery_info=None):
 class AmazonPollyProvider(Provider):
     """Amazon Polly speech api provider."""
 
-    def __init__(self, polly_client, config, supported_languages, all_voices):
+    def __init__(
+        self,
+        polly_client: boto3.client,
+        config: ConfigType,
+        supported_languages: list[str],
+        all_voices: dict[str, dict[str, str]],
+    ) -> None:
         """Initialize Amazon Polly provider for TTS."""
         self.client = polly_client
         self.config = config
         self.supported_langs = supported_languages
         self.all_voices = all_voices
-        self.default_voice = self.config[CONF_VOICE]
+        self.default_voice: str = self.config[CONF_VOICE]
         self.name = "Amazon Polly"
 
     @property
-    def supported_languages(self):
+    def supported_languages(self) -> list[str]:
         """Return a list of supported languages."""
         return self.supported_langs
 
     @property
-    def default_language(self):
+    def default_language(self) -> str:
         """Return the default language."""
-        return self.all_voices.get(self.default_voice).get("LanguageCode")
+        return self.all_voices[self.default_voice]["LanguageCode"]
 
     @property
-    def default_options(self):
+    def default_options(self) -> dict[str, str]:
         """Return dict include default options."""
         return {CONF_VOICE: self.default_voice}
 
     @property
-    def supported_options(self):
+    def supported_options(self) -> list[str]:
         """Return a list of supported options."""
         return [CONF_VOICE]
 
-    def get_tts_audio(self, message, language=None, options=None):
+    def get_tts_audio(
```

Comment:
Please check that keys may not exist in dicts and return `None, None` in that case.